stopes/pipelines/asr_bleu/requirements.txt

```diff
@@ -0,0 +1,7 @@
+fairseq==0.12.2
```

Comment:
we shouldn't have separate requirements, you can add a sub-group in the main pyproject.toml file if you need to (but probably the `speech` one already covers all that you need)

Reply:
Good point. Some dependencies such as sacrebleu aren't currently covered under `speech_mining`, so I added a new sub-group with all the dependencies that wouldn't be present in a base installation of stopes. Let me know if I should organize it in a different way.